src/test/java/dojo/portfolio/stringcalculator/WhenCalculatingResults.java

```diff
@@ -0,0 +1,85 @@
+package dojo.portfolio.stringcalculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@DisplayName("When calculating results with the string calculator")
+public class WhenCalculatingResults {
+    StringCalculator calculator;
+    @BeforeEach
+    void setUpCalculator(){
+     this.calculator = new StringCalculator();
+    }
+
+
+   @DisplayName("The calculator should return zero for empty string")
+    @Test
+    void shouldReturnZeroForEmptyStrings() throws illegalArgumentException {
+       double result = calculator.calculate("");
+       assertThat(result).isEqualTo(0.0);
+    }
+
+
+    @ParameterizedTest
+    @DisplayName("The calculator should return the number for a single integer number")
+    @CsvSource(
+           value = {
+            "1 = 1.0",
+    },
+            delimiterString = "="
+    )
+
+    void shouldReturnTheNumberForASingleNumber(String expression, double expected) throws illegalArgumentException {
+        double result = calculator.calculate(expression);
+        assertThat(result).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @DisplayName("The calculator should return the calculated results for two numbers")
+    @CsvSource(
```

Comment:
nice, csv source with '=' delimiter, that's cool

Reply:
Thanks